shacl12-ui/index.html

```diff
@@ -0,0 +1,842 @@
+          </dfn>
+          ,
+          <dfn data-lt="SHACL subclasses">
+            <a href="https://www.w3.org/TR/shacl/#dfn-shacl-subclass"
+              >SHACL subclass</a
+            >
+          </dfn>
+          ,
+          <dfn data-lt="shapes">
+            <a href="https://www.w3.org/TR/shacl/#dfn-shape">shape</a>
+          </dfn>
+          ,
+          <dfn data-lt="shapes graphs">
+            <a href="https://www.w3.org/TR/shacl/#dfn-shapes-graph"
+              >shapes graph</a
+            >
+          </dfn>
+          ,
+          <dfn data-lt="solutions">
+            <a href="https://www.w3.org/TR/shacl/#dfn-solution">solution</a>
+          </dfn>
+          ,
+          <dfn data-lt="subjects">
+            <a href="https://www.w3.org/TR/shacl/#dfn-subject">subject</a>
+          </dfn>
+          ,
+          <dfn data-lt="targets">
+            <a href="https://www.w3.org/TR/shacl/#dfn-target">target</a>
+          </dfn>
+          ,
+          <dfn data-lt="triples">
+            <a href="https://www.w3.org/TR/shacl/#dfn-rdf-triple">triple</a>
+          </dfn>
+          ,
+          <dfn
+            ><a href="https://www.w3.org/TR/shacl/#dfn-validation"
+              >validation</a
+            ></dfn
+          >
+          ,
+          <dfn data-lt="validation reports">
+            <a href="https://www.w3.org/TR/shacl/#dfn-validation-report"
+              >validation report</a
+            >
+          </dfn>
+          ,
+          <dfn data-lt="validation results">
+            <a href="https://www.w3.org/TR/shacl/#dfn-validation-results"
+              >validation result</a
+            >
+          </dfn>
+          ,
+          <dfn data-lt="validators">
+            <a href="https://www.w3.org/TR/shacl/#dfn-validators">validator</a>
+          </dfn>
+          ,
+          <dfn data-lt="values">
+            <a href="https://www.w3.org/TR/shacl/#dfn-value">value</a>
+          </dfn>
+          ,
+          <dfn data-lt="value nodes">
+            <a href="https://www.w3.org/TR/shacl/#dfn-value-nodes"
+              >value node</a
+            >
+          </dfn>
+          .
+        </p>
+      </section>
+
+      <section id="conventions">
+        <h3>Document Conventions</h3>
+        <p>
+          Within this specification, the following namespace prefix definitions
+          are used:
+        </p>
+        <table class="term-table">
+          <tr>
+            <th>Prefix</th>
+            <th>Namespace</th>
+          </tr>
+          <tr>
+            <td><code>rdf:</code></td>
+            <td><code>http://www.w3.org/1999/02/22-rdf-syntax-ns#</code></td>
+          </tr>
+          <tr>
+            <td><code>rdfs:</code></td>
+            <td><code>http://www.w3.org/2000/01/rdf-schema#</code></td>
+          </tr>
+          <tr>
+            <td><code>sh:</code></td>
+            <td><code>http://www.w3.org/ns/shacl#</code></td>
+          </tr>
```

Comment:
We could already add the `shui` prefix/namespace.
```suggestion
          <tr>
            <td><code>sh:</code></td>
            <td><code>http://www.w3.org/ns/shacl#</code></td>
          </tr>
          <tr>
            <td><code>shui:</code></td>
            <td><code>http://www.w3.org/ns/shui#</code></td>
          </tr>
```